src/server/graphql/httpGraphQLHandler.js

```diff
@@ -1,14 +1,13 @@
 import Schema from './rootSchema';
 import {graphql} from 'graphql';
+import {prepareClientError} from './models/utils';
 
 export default async (req, res) => {
   // Check for admin privileges
   const {query, variables, ...newContext} = req.body;
   const authToken = req.user || {};
-  const context = {authToken, context: newContext};
-  const result = await graphql(Schema, query, null, context, variables);
-  if (result.errors) {
-    console.log('DEBUG GraphQL Error:', result.errors);
-  }
-  res.send(result);
+  const context = {authToken, ...newContext};
+  const result = await graphql(Schema, query, null, context, variables)
+  const {error} = prepareClientError(result)
+  res.send({data: result.data, error})
```

Comment:
sending an `error` field in every payload is no good since we want it as short as possible. Rather it should rest on the client to know how to interpret the payload.